src/platform/src/main/java/org/geoserver/platform/ModuleStatus.java

```diff
@@ -14,7 +14,7 @@
  *
  * @author Morgan Thompson - Boundless
  */
-public interface ModuleStatus {
+public interface ModuleStatus extends ModuleCapabilities {
```

Comment:
This is weird, I would rather expect a module to have a list of capabilities, which by default (method) return an empty list.

Reply:
I would two methods to this interface with default methods:
- ``default List<ModuleCapability> getModuleCapabilities() {return Collections.getEmptyList();}``
- ``boolean hasCapability(ModuleCapability capability) {return false;}``

Reply:
fixed